app/jobs/process_push_update_job.rb

```diff
@@ -1,7 +1,7 @@
 class ProcessPushUpdateJob < ApplicationJob
   queue_as :default
 
-  def perform(_track)
+  def perform(track)
```

Comment:
How did this ever work before?

Reply:
It used to sync all tracks: https://github.com/exercism/v3-website/blob/master/app/jobs/process_push_update_job.rb#L9